app/utils/hash-utils.ts

```diff
@@ -54,9 +84,7 @@ const parseValue = (v: any) => {
   if (v === "undefined") return undefined;
   if (v === "[]") return [];
   if (v === "{}") return {};
-  if (v !== "" && !isNaN(Number(v))) {
-    return Number(v);
-  }
+  if (v !== "" && !isNaN(+v)) return +v;
```

Comment:
what if a we have a value that looks like a number but is a string ? "1234" ?

Reply:
Good point! https://github.com/visualize-admin/visualization-tool/pull/2239/commits/b6d257fa0ed9dfe551e755875b9ab7cd43edd605